### PR TITLE
Enable customized langtools test

### LINKED ISSUE #498: Enable a subset of langtools group
Enable and set up example of  a subset of langtools group

### PR DESCRIPTION
**Pull request by sophia-guo**
Closes #498 #493 

Signed-off-by: Sophia Guo <sophiag@ca.ibm.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
Test in my personal local jenkins as there are some GitSCM issues before( which seems now ok).

**Comment by sophia-guo** [comment]
@adamfarley when this is merged you should be able to run langtools javac tests either by personal build or grinder build.

File: openjdk_regression/playlist.xml [hunk 1]
```diff
@@ -25,6 +25,25 @@
...
 	$(Q)$(JTREG_TEST_DIR)$(D)$(CUSTOM_TARGET)$(Q); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>openjdk</group>
+		</groups>
+	</test>
+		<test>
```
Comment (llxia) [inline openjdk_regression/playlist.xml]: Please align `<test>`

File: buildenv/jenkins/JenkinsfileBase [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!groovy
 def getBuildList() {
-	def TESTPROJECTS = [system:'systemtest', perf:'performance', jck:'jck', external:'thirdparty_containers', functional: 'functional', openjdk:'openjdk_regression', jdk:'openjdk_regression', runtest:'', sanity:'', extended:'']
+	def TESTPROJECTS = [system:'systemtest', perf:'performance', jck:'jck', external:'thirdparty_containers', functional: 'functional', openjdk:'openjdk_regression',langtools:'openjdk_regression', jdk:'openjdk_regression', runtest:'', sanity:'', extended:'']
```
Comment (llxia) [inline buildenv/jenkins/JenkinsfileBase]: Can we just set BUILD_LIST to `openjdk_regression`?

File: openjdk_regression/playlist.xml [hunk 3]
```diff
@@ -25,6 +25,25 @@
...
+		<testCaseName>langtools_custom</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>   
+		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(TEST_RESROOT)$(D)jtreg$(D)lib$(D)jtreg.jar$(Q) \
+	$(JTREG_BASIC_OPTIONS) -vmoptions:$(Q)-Xmx768m $(JVM_OPTIONS)$(Q) \
+	-w $(Q)$(TEST_RESROOT)$(D)work$(Q) \
+	-r $(Q)$(TEST_RESROOT)$(D)report$(Q) \
+	-jdk:$(Q)$(JDK_HOME)$(Q) \
+	$(Q)$(JTREG_LANGTOOLS_TEST_DIR)$(D)$(LANGTOOLS_CUSTOM_TARGET)$(Q); \
```
Comment (llxia) [inline openjdk_regression/playlist.xml]: Currently grinder script does not take `LANGTOOLS_CUSTOM_TARGET`. How can user change this value in grinder?

File: openjdk_regression/playlist.xml (hunk 1 above)
Comment (sophia-guo) [reply openjdk_regression/playlist.xml]: Done

**Comment by sophia-guo** [comment]
Personal build:
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/698/
https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/700/

File: openjdk_regression/playlist.xml (hunk 3 above)
Comment (sophia-guo) [reply openjdk_regression/playlist.xml]: Thanks for pointing out. As discussed custom variable will be generated by custom target.

**Pull request merged**